SRC/classes/HelloWorldTestClass.cls

```diff
@@ -0,0 +1,17 @@
+@isTest 
+private class HelloWorldTestClass {
```

Comment:
try to name the test class same as the class and add test to it at the end,
Ex:  Class Name : MyHelloWorld 
       Test Class : MyHelloWorldTests